src/image.rs

```diff
@@ -333,7 +348,7 @@ impl<'a, I: GenericImage + 'a> Iterator for MutPixels<'a, I>
             // NOTE: This is potentially dangerous. It would require the signature fn next(&'a mut self) to be safe.
             // error: lifetime of `self` is too short to guarantee its contents can be safely reborrowed...
             let ptr = unsafe {
-                mem::transmute(tmp)
+                &mut *(tmp as *mut <I as GenericImage>::Pixel as *mut <I as GenericImage>::Pixel)
```

Comment:
I would prefer `transmute::<A, B>(tmp)` since it checks the size of the type. This is safer.